engine/src/main/java/com/datatorrent/stram/StramClient.java

```diff
@@ -448,6 +450,14 @@ public void startApplication() throws YarnException, IOException
       }
       String libJarsCsv = copyFromLocal(fs, appPath, localJarFiles.toArray(new String[]{}));
 
+      if (UserGroupInformation.isSecurityEnabled() && customKeystoreConfig != null && customKeystorePath != null) {
+        String[] customSSLConfArr = {customKeystoreConfig, customKeystorePath};
+        String customSSLFileConf = copyFromLocal(fs, appPath, customSSLConfArr);
+        String configFileName = (new java.io.File(customKeystoreConfig)).getName();
+        dag.getAttributes().put(Context.DAGContext.STRAM_HTTP_CUSTOM_CONFIG, configFileName);
```

Comment:
Regarding STRAM_, we may also want to look at consistency. There is already a STRAM_HTTP_AUTHETICATION property. Maybe that shouldn't have been named with STRAM_ prefix in the first place and in retrospect, MASTER_ may have been a better prefix as there is a MASTER_MEMORY_MB. I still think just HTTP_ is vague and ambiguous. Maybe a better way to name this is to refer to the functionality, this attribute is for, such as MANAGEMENT_HTTP_* and internally it gets applied to the right component, in this case stram.

Regarding CUSTOM, I am fine with removing it or going with an alternative.